frontends/web/src/components/banner/banner.tsx

```diff
@@ -44,11 +45,11 @@ function Banner({ banner, i18n, t }: TProps) {
     return null;
   }
   return banner && (
-    <Status dismissible="" type="warning">
+    <Status dismissible={banner.dismissible ? banner.id : ''} type="warning">
```

Comment:
so banner.id was unused?

Reply:
yeah, I think so

Reply:
it was used, see here:

https://github.com/digitalbitbox/bitbox-wallet-app/commit/a0601e2cb4330b8047ca15ff95597b1c22a69d9c#diff-694b6e6a9809368b1c603dba8dc008244ccb8e76febfb19ab760258dbe329ab8R47

makes sense to use the key as it used to be like this:

`dismissible={`banner-${msgKey}-${banner.id}`}` so the ID is scoped.

Reply:
Good point, thanks! Done